draft-seemann-quic-reliable-stream-reset.md

```diff
@@ -94,18 +95,18 @@ RELIABLE_RESET_STREAM Frame {
 
 RELIABLE_RESET_STREAM frames contain the following fields:
 
-Stream ID:  A variable-length integer encoding of the stream ID of
+Stream ID:  A variable-length-encoded integer encoding of the stream ID of
```

Comment:
This was just copied from the RFC, so I'd prefer to keep it as is: https://datatracker.ietf.org/doc/html/rfc9000#section-19.4

```suggestion
Stream ID:  A variable-length integer encoding of the stream ID of
```